charon/config.py

```diff
@@ -57,10 +63,15 @@ def validate(self) -> bool:
         if self.__root_ca and not os.access(self.__root_ca, os.R_OK):
             logger.error("The root ca file is not valid!")
             return False
+        if self.__quay_radas_registry_config and not os.access(
+            self.__quay_radas_registry_config, os.R_OK
+        ):
+            logger.error("The quay registry config for oras is not valid!")
+            return False
```

Comment:
Maybe this should not fail the radas config, but just give a warn message and leave this registry_config as None, which means we will consider it as a public registry without authentication.

Reply:
@ligangty done, https://github.com/Commonjava/charon/pull/294/commits/947f23ce5434e0131c32e3825e8f566eca2f9753